clib/src/main/resources/scala-native/epoll.c

```diff
@@ -0,0 +1,26 @@
+#ifdef __linux__
```

Comment:
For the benefit of future readers, there is no need for
provisional compilation "compile only if used here".
These file are in `libc` or in the future below it.
Such files are "always" compiled, perhaps with os guards.

Yes, that introduces empty files and costs CPU cycles
for things never used. It is however, the state of the state.